src/__snapshots__/compile.test.ts.snap

```diff
@@ -270,4 +294,4 @@ Array [
 ]
 `;
 
-exports[`when targeting SDK 2.0 does not allow JSON imports 1`] = `<cwd>/src/__test__/compile/data.json: Unexpected token, expected ; (2:7)`;
+exports[`when targeting SDK 2.0 does not allow JSON imports 1`] = `"Unexpected token (Note that you need rollup-plugin-json to import JSON files)"`;
```

Comment:
No path is printed? And what do we want to do about the note? It'll be confusing for our users since they have no control over rollup plugins.

Reply:
Turns out it wasn't rollup but instead rollup-plugin-babel that was emitting this error. Managed to keep the upgrades by re-enabling it's old behaviour whereby it'd try to compile JSON files.